ocaml/xenopsd/c_stubs/xenctrlext_stubs.c

```diff
@@ -113,14 +113,11 @@ CAMLprim value stub_xenctrlext_get_runstate_info(value xch_val, value domid)
 	CAMLparam2(xch_val, domid);
 #if defined(XENCTRL_HAS_GET_RUNSTATE_INFO)
 	CAMLlocal1(result);
-	xc_runstate_info_t info = { 0 };
```

Comment:
What's wrong with keeping this initialization of `info` to `{ 0 }`? otherwise it's going to be indeterminate and might not be initialized in the xenctrl call.